packages/database/src/migrations/20200929223403-AddTileSetsToProject-modifies-scheme.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import { TupaiaDatabase } from '@tupaia/database';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function () {
+  const db = new TupaiaDatabase();
```

Comment:
Is there a reason we are doing the `new TupaiaDatabase()` here instead of doing `exports.up = async function (db) { ... }` ?

Does it trigger some functions?

Reply:
To be honest, this is my first time writing a migration in Tupaia so I just looked up another example for reference and copied the syntax. I went back and looked at some more examples now and it seems that different ones are done different ways. I have updated it to the way you mentioned though as it makes more sense to me.